keras/preprocessing/image.py

```diff
@@ -253,8 +253,8 @@ def array_to_img(x, data_format=None, scale=True):
         x = x + max(-np.min(x), 0)
```

Comment:
This line already creates a copy of the array. Therefore subsequent lines do not modify the original array in place, they modify the copy.